onnxruntime/core/providers/tensorrt/tensorrt_execution_provider.cc

```diff
@@ -3243,6 +3268,14 @@ Status TensorrtExecutionProvider::CreateNodeComputeInfoFromGraph(const GraphView
       if (trt_state->auxiliary_streams >= 0) {
         LOGS_DEFAULT(WARNING) << "[TensorRT EP] Auxiliary streams can only be set on TRT 8.6 onwards!";
       }
+#endif
```

Comment:
Looks like these lines are identical to 2765. Can this be refactored into a common function?

Reply:
Hmm, those logging macros eventually use `ORT_WHERE` which pulls in `__LINE__` and `__FUNCTION__`